controlpanel/api/models/apps3bucket.py

```diff
@@ -24,9 +22,19 @@ class Meta:
         unique_together = ('app', 's3bucket')
         ordering = ('id',)
 
-    def aws_name(self):
+    @property
+    def iam_role_name(self):
         return self.app.iam_role_name
 
     def __repr__(self):
         return f'<AppS3Bucket: {self.app!r} {self.s3bucket!r} {self.access_level}>'
 
+    def grant_bucket_access(self):
+        cluster.App(self.app).grant_bucket_access(
+            self.s3bucket.arn,
+            self.access_level,
+        )
+
+    def revoke_bucket_access(self):
+        cluster.App(self.app).revoke_bucket_access(self.s3bucket.arn)
```

Comment:
nice ☺️

Reply:
oh you added these to a `cluster.App` class instead of having something like `cluster.AppS3Bucket`.
Yeah makes sense as it's the subject anyway 👍